L55-objc-global-interceptor.md

```diff
@@ -0,0 +1,60 @@
+Title
+----
+* Author(s): muxi
+* Approver: psrini
+* Status: In Review
+* Implemented in: Objective-C
+* Last updated: 2019-06-17
+* Discussion at: https://groups.google.com/forum/#!topic/grpc-io/tRKcPcfQKi4
+
+## Abstract
+
+This proposal is about global interceptor for gRPC Objective-C library.
+
+## Background
+Global interceptor refers to interceptors that run across all the calls in the same process, despite whether it is explicitly added to a particular call in its call options. It is a powerful feature for use cases such as global logging. The feature has been implemented in gRPC libraries of other languages.
+
+
+### Related Proposals: 
+* L50 - gRPC Objective-C Interceptor
+
+## Proposal
+
+### Global interceptor interface
+
+The proposed gRPC Objective-C library's global interceptor interface is as follows:
+
+```objectivec
+@interface GRPCCall2 (Interceptor)
+
+/**
+ * Register a global interceptor's factory in the current process. Only one interceptor can be
+ * registered in a process. If another one attempts to be registered, an exception will be raised.
+ */
++ (void)registerGlobalInterceptor:(nonnull id<GRPCInterceptorFactory>)interceptorFactory;
```

Comment:
Do other langs have unregister API, in case app wants to remove/replace the global interceptor on-the-fly?

Reply:
Asked Java and C++ and they do not have it, with rationale that it makes the hack (global interceptor) more complicated.